baseplate/lib/events.py

```diff
@@ -93,10 +95,18 @@ class EventQueue(ContextFactory, config.Parser, Generic[T]):
 
     """
 
-    def __init__(self, name: str, event_serializer: Callable[[T], bytes]):
-        self.queue = MessageQueue(
-            "/events-" + name, max_messages=MAX_QUEUE_SIZE, max_message_size=MAX_EVENT_SIZE
-        )
+    def __init__(
+        self,
+        name: str,
+        event_serializer: Callable[[T], bytes],
+        queue_type: QueueType = QueueType.POSIX,
```

Comment:
You also need to add this parameter to the doc string.

Reply:
hmm, I dont understand how this would work. Anywhere anyone instantiates an `EventQueue` they would have to provide the queue? I feel like part of using the `EventQueue` is that it creates the queue for you, if we changed this it would require Baseplate users to change their applications, unless I'm misunderstanding